tests/unit/fileSyncerManager/fileSyncerManager.spec.ts

```diff
@@ -70,6 +84,16 @@ describe('fileSyncerManager', () => {
       expect(taskHandlerMock.reject).toHaveBeenCalled();
     });
 
+    it(`When get or post file throws unknown error, catches the error`, async () => {
```

Comment:
typo- catch anot catches

Reply:
Why? third body, it is supposed to be "catches"